policies/bundles/DocumentCSFAContentValid/input.json

```diff
@@ -0,0 +1,16 @@
+{
+    "filetype": "JSON",
+    "id": "https://xx.yy.zz/XXX",
+    "name": "ServiceMetadataDocument1",
```

Comment:
is the name a human readable name or the file name? I am not sure how we are discovering this